<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wgt</class>
 <widget class="QWidget" name="Wgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="host">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>10.8.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>57</width>
     <height>27</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>6667</number>
   </property>
  </widget>
  <widget class="QPushButton" name="connect">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>64</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="channel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>#chan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="join">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Join channel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="channelList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel list</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nick">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>z80</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nick</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>271</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Client list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="isConnected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Is connected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="isJoined">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Is joined</string>
   </property>
  </widget>
  <widget class="QPushButton" name="status">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="to">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>280</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>z80</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="msg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>310</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Test message</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Msg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dccRequest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DCC request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dccSend">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>340</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DCC send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dccFile">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>340</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DCC file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
